index.html

```diff
@@ -8,12 +8,34 @@
 
 	<title>Nasz prawie autorski design</title>
 
+
+	<script src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
+
+
 	<!-- Loading third party fonts -->
 	<link href="http://fonts.googleapis.com/css?family=Roboto:300,400,700|" rel="stylesheet" type="text/css">
 	<link href="fonts/font-awesome.min.css" rel="stylesheet" type="text/css">
 
 	<!-- Loading main css file -->
 	<link rel="stylesheet" href="style.css">
+	<script  type="text/javascript">
+		var data_url="https://api.openweathermap.org/data/2.5/weather?q=Warsaw,pl&APPID=7cad625ece0a657d34524777c3f14cfa";
+
+		getWeather(function (data) {
+			var temp = data.main.temp-273.15
+			var weather_html = (temp).toPrecision(2)+"<sup>o</sup>C";
+			document.getElementById('opady').innerHTML = weather_html;
+		});
+		// function to use jsonp to get weather information
```

Comment:
Ten kod nie ma nic wspólnego z _jsonp_, ale to dobrze. **AJAX** jest zdecydowanie lepszy.